### ISSUE TITLE
Recording doesn't work

### ISSUE THREAD

**Issue by lonkaars** [phisch/giph#24]
Not sure if #6 is related, but I get an error saying ffmpeg can't record. After some investigation, I fixed it by changing

```bash
ffmpeg=(ffmpeg -f x11grab -s "$width"x"$height" -i "$DISPLAY.0+$x,$y")
```
to
```bash
ffmpeg=(ffmpeg -f x11grab -s "$width"x"$height" -i "$DISPLAY+$x,$y")
```

in `function record()` on line 347

On my system, $DISPLAY is set to `:0.0` which is set on startup in my ~/.xprofile by running `systemctl --user import-environment DISPLAY`. I'm not sure if this is standard, or if something like `:0` is standard, so I'm not sure if I should fork and send a PR to fix this.

**Comment by phisch** [comment]
@lonkaars apparently some older systems use `:0.0` while most systems use `:0`. Which distro are you using?

But that's not really important since ffmpeg ignores the `.0`, which it apparently only supports for legacy reasons. So, removing the `.0` from https://github.com/phisch/giph/blob/master/src/giph#L348 as you mentioned works in both cases.

I will fix this for the next release!

**Comment by lonkaars** [comment]
I'm on Arch Linux with kernel 5.12.10. It resolved the variables into something like `-i :0.0.0+`... which is why ffmpeg crashed.

Sort of unrelated, but could you remove `-loglevel quiet` from ffmpeg's args when running giph with verbosity >= 1?

Thanks for fixing anyways!

**Comment by phisch** [comment]
`Arch Linux with kernel 5.12.10` weird. I have the same. No idea why yours appends the `.0` and mine doesn't. Anyway, i'll remove it, and this issue should be solved.

 Log levels are described in the manpage, and debug is the only level that makes sense to me for ffmpeg output. https://github.com/phisch/giph/blob/master/man/giph.1#L50
